packages/kbn-monaco/src/esql/lib/ast/validation/validation.test.ts

```diff
@@ -845,6 +884,16 @@ describe('validation logic', () => {
       []
     );
 
+    for (const field of fieldTypes) {
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field IS NULL`, []);
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field IS null`, []);
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field is null`, []);
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field is NULL`, []);
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field IS NOT NULL`, []);
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field IS NOT null`, []);
+      testErrorsAndWarnings(`from a | where ${camelCase(field)}Field IS not NULL`, []);
```

Comment:
Does ES|QL support mixed capitalization? (e.g. `Is nOt NuLL`)

Reply:
It does, @dej611  why our validator complains? This is valid 🤔 

<img width="495" alt="image" src="https://github.com/elastic/kibana/assets/17003240/61c242e0-960c-40f6-ab46-1f430c4ce788">

Reply:
Yes it does, although it seems weird. :D

Reply:
I've added a test with the operator in camel case, as @drewdaemon wrote, but it works.
In general operations/functions are all lowerCased in the AST pipeline (although the original case is preserved int he raw `text`).